pkgs/applications/audio/rhythmbox/default.nix

```diff
@@ -21,7 +21,7 @@
 , libgpod
 , libmtp
 , lirc
-, brasero
+, brasero-unwrapped
```

Comment:
Add a comment you're using brasero-unwrapped so you don't pull in libdvdcss which isn't used by rhythmbox.